app/mobile-content/exhibits/shared/exhibit.model.ts

```diff
@@ -17,8 +17,8 @@ export class Exhibit {
   constructor(public id: number,
               public name: string,
               public description: string,
-              public latitude: number,
```

Comment:
Can we maybe still use a number internally, despite the input field returning a string? Feels cleaner to have a number here.